mineinabyss-features/src/main/kotlin/com/mineinabyss/misc/MiscListener.kt

```diff
@@ -0,0 +1,19 @@
+package com.mineinabyss.misc
+
+import org.bukkit.entity.ItemFrame
+import org.bukkit.entity.ThrownPotion
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.entity.ProjectileHitEvent
+import org.bukkit.potion.PotionEffectType
+
+class MiscListener : Listener {
+    @EventHandler
+    fun ProjectileHitEvent.onDouseItemFrame() {
+        val entity = entity as? ThrownPotion ?: return
+        if (hitEntity !is ItemFrame) return
+        if (entity.potionMeta.basePotionData.type.effectType != PotionEffectType.INVISIBILITY) return
+        hitEntity?.location?.getNearbyEntitiesByType(ItemFrame::class.java, 1.0)?.forEach { frame -> frame.isVisible = false
+        }
+    }
+}
```

Comment:
seems someone reused the same branch lol. No worries it should not cause issues since its identical but